safety_shield/src/safety_shield.cc

```diff
@@ -451,12 +448,57 @@ Motion SafetyShield::computesPotentialTrajectory(bool v, const std::vector<doubl
     } 
     goal_motion.setTime(potential_path_.getPhase(3));
     return goal_motion;
-  } catch (const std::exception &exc) {
+  } catch (const std::exception& exc) {
     spdlog::error("Exception in SafetyShield::computesPotentialTrajectory: {}", exc.what());
     throw exc;
+    return Motion();
   }
 }
 
+bool SafetyShield::planPFLFailsafe(double a_max_manoeuvre, double j_max_manoeuvre) {
+  // Calculate maximal Carthesian velocity in the short-term plan
+  double s_d, ds_d, dds_d;
+  // Calculate goal
+  bool is_under_iso_velocity = false;
+  // v_max is maximum of LTT or STP and vel_s_dot is how much path velocity needs to be scaled to be under v_iso
```

Comment:
what is `vel_s_dot`? Do you mean `v_limit`?

Reply:
yes, you are right.